### ISSUE TITLE
kor doesn't output in JSON format

### ISSUE THREAD

**Issue by doryer** [yonahd/kor#35]
**Describe the bug**
kor cli doesn't output in JSON format while in table mode it is generating output normally

**To Reproduce**
Running: 
kor all -n default --output json

**Expected behavior**
Output in JSON format

**Screenshots**
<img width="596" alt="Screen Shot 2023-08-29 at 12 41 14" src="https://github.com/yonahd/kor/assets/94246801/3047adb0-c3f3-492c-80d9-0d35ed6cf233">

Mac os M1
Kor version: 0.1.9

**Comment by Yuni-sa** [comment]
hi, it looks like in all of the GetUnusedJson() functions for example `GetUnusedDeploymentsJSON()` , the function returns the correct response but does not print it. @yonahd

**Comment by yonahd** [comment]
Yes. This was developed to return a json to other code using kor.

We should support it as output as well.

**Comment by Yuni-sa** [comment]
maybe it should be added in the command files like so: 
```go
if outputFormat == "json" {
	if jsonResponse, err := kor.GetUnusedDeploymentsJSON(namespace, kubeconfig); err != nil {
		fmt.Println(err) // also add some formatting
	} else {
		fmt.Println(jsonResponse)
	}
} else {
	kor.GetUnusedDeployments(namespace, kubeconfig)
}
```
and in that way it doesn't print anything unnecessary when returning the response to code.

**Comment by yonahd** [comment]
@Yuni-sa That should work.
I see that also a couple of the resources are printing the data e.g. https://github.com/yonahd/kor/blob/main/pkg/kor/hpas.go#L127
These should be removed

Can you open a MR for this?

**Comment by Yuni-sa** [comment]
> @Yuni-sa That should work. I see that also a couple of the resources are printing the data e.g. https://github.com/yonahd/kor/blob/main/pkg/kor/hpas.go#L127 These should be removed
> 
good catch! can you asign me to the issue? @yonahd
